shard-operator/src/main/java/com/redhat/service/bridge/shard/operator/networking/KubernetesNetworkingService.java

```diff
@@ -46,8 +46,9 @@ public EventSource buildInformerEventSource(String component) {
     @Override
     public NetworkResource fetchOrCreateNetworkIngress(BridgeIngress bridgeIngress, Service service) {
         Ingress expected = buildIngress(bridgeIngress, service);
-
         Ingress existing = client.network().v1().ingresses().inNamespace(service.getMetadata().getNamespace()).withName(service.getMetadata().getName()).get();
+        LOGGER.debug("Expected Ingress: {}", expected);
+        LOGGER.debug("Existing Ingress: {}", existing);
```

Comment:
remove?

Reply:
Well.. it was useful for me.. and it is at `debug` level (so not logged most of the time).. but if you want it gone it can.

Let me know if you have a strong preference.

Reply:
It's not really useful since the ingress is not expected to change unless in very few rare cases. I'd remove those if you are ok

Reply:
OK, you have much more experience that I do in this area so I will remove.